roxctl/central/generate/generate.go

```diff
@@ -232,12 +240,23 @@ func createBundle(logger logger.Logger, config renderer.Config) (*zip.Wrapper, e
 
 // OutputZip renders a deployment bundle. The deployment bundle can either be
 // written directly into a directory, or as a zipfile to STDOUT.
-func OutputZip(logger logger.Logger, io io2.IO, config renderer.Config) error {
+func OutputZip(logger logger.Logger, io io2.IO, config *renderer.Config) error {
```

Comment:
I agree, indeed, introducing a pointer here is a case of premature optimization. I'll roll back this particular change.

Reply:
On the other hand, `OutputZip` would add some values to `config.SecretsByteMap`, which will modify the original config anyway. So it could actually make sense to highlight the possible side effect with the pointer.